jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/annotations/ResourceAnnotation.java

```diff
@@ -1,23 +1,22 @@
 package io.openliberty.sample.jakarta.annotations;
 
 import jakarta.annotation.Resource;
+import jakarta.annotation.Resources;
 
-@Resource(type = Object.class, name = "aa")
+@Resources ({ @Resource(name = "aaa"), @Resource(type = Object.class) })
```

Comment:
A couple of things here:
1. You may want to format this file.
2. Why replace @Resource for @Resources? That could be used for a positive test where we check that no diagnostics are generated because everything looks fine.
3. I would suggest that you create a ResourcesAnnotation class and add a similar set of code for a similar set of diagnostics/code-action tests including a positive test.